### ISSUE TITLE
information on the step argument "closure"

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [janEbert/PyTorch-VeLO#2]
Hi,

I've been trying to use VeLO in my code recently and I get an error because I'm not giving argument "closure" to .step().

I was wondering if you could give some information on exactly what I should give as closure as I couldn't find information in the code directly :)

Thank you and have a nice day!

**Comment by thiswillbeyourgithub** [comment]
Thanks to GPT4 I was able to figure it out.

optimizer.step() takes as argument a function, that takes no argument and returns either a 2 tuple (loss, model_state) or just the loss as a scalar

Thanks a lot for making this!

**Comment by janEbert** [comment]
Hey, glad you were able to figure this out! :)
Documentation is really lacking, thank you for pointing this out. Please also feel free to share you experience using this. I found the JAX/PyTorch interop to be really slow, but maybe you have a different experience.

**Comment by thiswillbeyourgithub** [comment]
Well my standards are pretty low so it seems to be working and it's already awesome :)! What's annoying is (if i'm not mistaken) the declaration time : creating the optimizer seems to be taking up to 10s or something. Also I couldn't find a way to suppress the many logging prints caused by loading the checkpoint. And it was far from easy to manage to find the right jax/jaxlib package to go with my torch setup.
